scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt

```diff
@@ -79,6 +79,18 @@ class ScannerIntegrationFunTest : WordSpec({
                     matchExpectedResult(expectedResultFile)
         }
     }
+
+    "scan() a subset of the packages corresponding to a single VCS" should {
+        "return the expected ORT result" {
```

Comment:
IIUC, this test does not pass yet and should be disabled here, and enabled in the next commit.

Reply:
The test does pass. To be sure, I just verified it again. Note that the expected result is updated in the next commit to illustrated the effect of the implemented filtering.